### DISCUSSION TITLE
Disable update notifications?

### DISCUSSION THREAD

**Discussion by TheQuickFox** [rocksdanister/lively#1694] [General]
I'm a happy user of Lively Wallpaper. But I got a question: Is it possible to disable update notifications?
I prefer to update my software manually at the time of my own preference. If it is not possible now, can you add an option to update manually with no update notifications?

When I close an update notification now it comes back every time I start my computer.

**Comment by rocksdanister** [comment]
It is not possible right now, how will the user know there is update if notification is disabled?

The Microsoft Store version of Lively has no update checker built in, store will silently update or manually you can update by opening store -> Library -> Get Updates.

**Reply by TheQuickFox** [reply]
Suggestion: 
You could add a button / menu in the UI to check for updates.

Button: Check for updates now.

Option: Automatically check for updates (Default: YES, I would set to NO)
Option: Automatically install updates (Default: No)
